GraphQL.Net/GraphQLTypeBuilder.cs

```diff
@@ -129,6 +129,12 @@ public void AddAllFields()
                 _type.OwnFields.Add(CreateGenericField(prop));
         }
 
+        public void AddAllFields(Dictionary<string, string> aliasDictionary)
+        {
+            foreach (var key in aliasDictionary.Keys)
+                _type.OwnFields.Add(CreateGenericField(key, aliasDictionary[key]));
+        }
+
         // unsafe generic magic to create a GQLField instance
         private GraphQLField CreateGenericField(PropertyInfo prop)
```

Comment:
Could you change this method to take an optional `string alias` parameter? Then if that parameter is null, populate it with `prop.Name.ToCamelCase()`. Then in the below method you can just get the prop info for `sourceProp` and delegate to this method.